src/persistence/db/identity_chain.rs

```diff
@@ -21,6 +32,24 @@ impl SurrealIdentityChainStore {
     pub fn new(db: Surreal<Any>) -> Self {
         Self { db }
     }
+
+    async fn create_block(&self, query: &str, entity: IdentityBlockDb) -> Result<()> {
+        let _ = self
+            .db
+            .query(query)
+            .bind(("table", Self::TABLE))
```

Comment:
maybe this strings in some enum? and we will call it everywhere, where we need it